### PR TITLE
exposing mapping_distance in travel builder

### PR DESCRIPTION
**Pull request by knaaptime**
expose mapping_distance

this just allows you to pass a mapping_distance through to pandana's snapping mechanism but it induced some formatting changes

### CODE REVIEW

**Comment by jGaboardi** [comment]
Did I jump the gun on merging #755 ?

**Comment by knaaptime** [comment]
not really i pushed this the instant you hit merge. I think this can just go in

**Comment by knaaptime** [comment]
and it switches the docstrings, which i think is more useful. The private funcs had the more descriptive docs last time

**Review by jGaboardi — APPROVED** [review]

File: libpysal/graph/base.py [hunk 1]
```diff
@@ -1456,18 +1456,21 @@ def build_h3(cls, ids, order=1, weight="distance"):
...
             geodataframe representing observations which are snapped to the nearest
-            node in the pandana.Network. If passing polygon geometries, the spatial
-            support will be reduced to Points (via centroid) before snapping.
+            node in the pandana.Network. CRS should be the same as the locations
+            of ``node_x`` and ``node_y`` in the pandana.Network (usually 4326 if network
+            comes from OSM, but sometimes projected to improve snapping quality).
         network : pandana.Network
             pandana Network object describing travel costs between nodes in the study
-            area
+            area.  See <https://udst.github.io/pandana/> for more
```
Comment (martinfleis) [inline libpysal/graph/base.py:1473]: We should probably use intersphinx here to link to the class but that is a just a minor detail.

**Pull request merged**